src/styles/price.scss

```diff
@@ -0,0 +1,8 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: $defaultWidth;
```

Comment:
The variable `$defaultWidth` is used but not defined. Ensure it is defined in your SCSS files to avoid compilation errors.